legion/engine/llri-dx/detail/instance.cpp

```diff
@@ -73,7 +73,7 @@ namespace LLRI_NAMESPACE
             auto* output = new Instance();
             UINT factoryFlags = 0;
 
-            for (uint32_t i = 0; i < desc.numExtensions; i++)
+            for (size_t i = 0; i < desc.numExtensions; i++)
```

Comment:
i see that desc.numExtensions is uint32_t, this is technically the same as size_t on 32bit platforms, but not on 64bit platforms. tbh generally for sizes and counts you might just wanna stick with size_t. specifying anything smaller than 64 won't give a speedup or noticeable size difference due to alignment and padding.